README.md

```diff
@@ -1 +1,54 @@
+
+## Significance 
+
+These RFCs are in practice only a signaling mechanism to determine and indicate the Fellowship's design and architecture preferences and to coordinate discussion and social consensus on architectures and designs according to open-source principles.
+
+The Fellowship holds only the powers vested in it by Polkadot's governance, which are limited to the expression of expert opinion and the ability to move proposals to more lenient governance tracks when necessary. It is not an arbiter of the "correctness" of any particular runtime or node implementation, and the practical meaning of these RFCs follows as a consequence of its limited powers. 
+
+For any RFC concerning runtime logic or interfaces, the Fellowship's capabilities are bounded by relay-chain governance, which is the ultimate decider of what code is adopted for block processing. As such, these RFCs are only loosely binding - the chains' governance has no obligation to accept the features as implemented and may accept features which have not gone through the RFC process. When it comes to node-side areas of expertise, the Fellowship's vote is more strongly binding, as the governance systems of the chains can't determine the environment the runtime is executed within, and in practice all node implementations should conform to some foundational standards in order to communicate.
+
+Merged RFCs are only an indication of support for a specific design, not a commitment to an implementation of a feature on any particular timeframe or roadmap ordering.
+
+## Process
+
+The RFC process is open to all contributors. Anyone may open an RFC or provide comments on open RFCs.
+
+To open an RFC, follow these steps:
+  * Copy the `0000-template.md` file into the `text` folder and rename to match the title of the RFC
+  * Fill out the RFC template and open a PR.
+  * Rename the file to correspond to the GitHub pull request number and update the "RFC PR" field in the file.
+
+The Fellowship will decide, via an on-chain voting mechanism including members III-Dan or above, when to approve and merge RFCs. It does so by issuing an on-chain remark with the body `RFC_APPROVE(xxxx, h)` from the `Fellows` origin on the Polkadot Collectives blockchain, where xxxx is the number of the RFC and h is the blake2-256 hash of the raw proposal text. Once this remark has been made, the PR can be merged. This on-chain process is designed to be resilient to where the RFCs are hosted and in what format, so it can be migrated away from GitHub in the future. The fellowship should not approve more than one RFC with the same number.
+
+The Fellowship may also decide to reject an RFC by issuing a remark with the text `RFC_REJECT(xxxx, h)`. This is a formality to provide clarity on when PRs (or their analogue on non-GitHub platforms) may be closed. PRs may be closed by their author, as well. PRs may be closed when sufficiently stale, as well - after a period of 1 year without acceptance.
```

Comment:
I realize that it's not the most fun problem to solve, but if I had to name one problem in the Polkadot ecosystem it's: too many ideas that either abandoned or not moving.

If the point of the fellowship is to be the "technical masters" of Polkadot, then they should fulfill their role of "opinion givers", it would in my opinion it would be preferably to have some kind of deadline after which fellowship members _must_ vote. If the fellowship completely abstains from voting, it means that they (as a group) doesn't fulfill this role of giving opinions.

Also, in my opinion it should be mentioned that implementers should make a reasonable effort to implement the accepted RFCs. After all, if an implementer decides to forever post-pone implementing an RFC, it means that the opinion of the fellowship as no value.

Reply:
>  It would in my opinion be preferably to have some kind of deadline after which fellowship members must vote

Disagree because,
a) because it's not enforceable, has no consequences for not meeting that requirement
b) either you can have open RFCs that anyone can submit OR required voting
c) some ideas are just not worth spending time on

Reply:
> My opinion is if a fellow is completely idle, they should be removed from the fellowship. Or is being a fellowship member some kind of irrevocable aristocratic title that grants you a rent for life for doing nothing?

No, to stay in the fellowship you need to prove you're active. Any serious RFC should be able to find a champion who will create a referendum on-chain, but rules vs. social norms are very different things.

If lots of serious RFCs are falling through the cracks I'd be the first person to push more fellows to be active.

> to me this feels like you want to solve a non-existing problem.

Likewise - therefore it makes more sense now to have fewer rules and then add more based on the problems that do exist.